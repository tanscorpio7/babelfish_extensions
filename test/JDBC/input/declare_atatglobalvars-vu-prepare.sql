create table t1_declare_atatglobalvars (a int)
go
insert t1_declare_atatglobalvars values (1), (2)
go
create type tt_declare_atatglobalvars as table (a int)
go
