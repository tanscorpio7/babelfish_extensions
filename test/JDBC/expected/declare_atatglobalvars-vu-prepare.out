create table t1_declare_atatglobalvars (a int)
go
insert t1_declare_atatglobalvars values (1), (2)
go
~~ROW COUNT: 2~~

create type tt_declare_atatglobalvars as table (a int)
go
